python/Makefile

```diff
@@ -0,0 +1,32 @@
+schemas = $(shell find ../jsonschema -name "*.json")
+
+.DEFAULT_GOAL = help
+
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+HERE := $(dir $(MKFILE_PATH))
+
+help: ## Show this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make <target>\n\nWhere <target> is one of:\n"} /^[$$()% a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+generate:
+	datamodel-codegen \
```

Comment:
This will not work. But CI passed because the Makefile in root does not invoke code gen Python.

Ideally python uses the same approach as the other languages. This ensures that the code can be generated without installing every language.

Further more the generated models should be checked in. This ensures that all code changes can be inspected and that the code gen process is not required to build a specific language.

Reply:
Other language realizations use additional tools. For Python such a tool is `datamodel-codegen` that needs Python environment (So a language interpreter is needed here not for generation itself but for launching a tool. ) . This tool targets the most specific task - generating Python models from json schemas. I insist on not competing with specific tools with own just written solution.

Reply:
Okay that's fair. I'll take a look at the build process. Perhaps we can fit in as is.

Reply:
Added dependency on python so genenerate-all flow works.

Reply:
As discussed directly. You can stub out the make file and invoke the code generation directly from the build process.

Reply:
Done